app/assets/css/main.css

```diff
@@ -508,6 +508,126 @@ Yeoman Team
   border-radius: 50%;
 }
 
+/*
+Circle loading indicator from SpinKit
+https://github.com/tobiasahlin/SpinKit
+*/
+
+.sk-circle {
+  margin: 40px auto;
+  width: 80px;
+  height: 80px;
+  position: relative; }
```

Comment:
Can you slightly adapt the style of the new CSS code to match the previous one?

- Put closing braces on a new line
- Leave a space between rules